src/js/containers/Menu.js

```diff
@@ -22,6 +23,11 @@ const mapStateToProps = (state) => {
             if (ddState === true && menuDepth === 1) {
                 enabled = false;
             }
+
+            if (command.subMenu.length === 0) {
+                // Found and empty submenu, ask app to send add commands
+                UIController.onUpdateSubMenu(activeApp, command.menuID);
+            }
```

Comment:
I don't see this trigger go off, `OnUpdateSubMenu` is not sent when an empty submenu is shown or opened

Reply:
Should be triggered when one level higher than an empty sub menu. Idea is to have the items populated that are one level deeper than the current menu depth.

Reply:
Using the same code above, here is the onsubMenu triggered after opening the main menu:

![Screen Shot 2020-08-05 at 12 18 21 PM](https://user-images.githubusercontent.com/12038416/89437593-c6ee1c00-d715-11ea-89d0-ecc919d3ac0b.png)

Reply:
Does this still work for nested submenus on your end? I'm still seeing the issue